Source/WebCore/page/scrolling/mac/ScrollerMac.mm

```diff
@@ -332,6 +332,7 @@ - (void)invalidate
     m_scrollerImpDelegate = adoptNS([[WebScrollerImpDelegateMac alloc] initWithScroller:this]);
     m_scrollerImp = [NSScrollerImp scrollerImpWithStyle:nsScrollerStyle(m_pair.scrollbarStyle()) controlSize:nsControlSizeFromScrollbarWidth(m_pair.scrollbarWidthStyle()) horizontal:m_orientation == ScrollbarOrientation::Horizontal replacingScrollerImp:nil];
```

Comment:
If we called `setScrollerImp()` here that would take care of the `updateMinimumKnobLength() call.

Reply:
Fixed.